maintainers/maintainer-list.nix

```diff
@@ -7477,6 +7477,12 @@
     githubId = 42433779;
     name = "Rémy Grünblatt";
   };
+  rguevara84 = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add rguevara84``.